client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/CustomEventLogListener.java

```diff
@@ -238,6 +238,30 @@ private void expectTimeOrdered(CustomEvent before, CustomEvent after) {
       public DynamicCustomEventWithStateLogListener(RemoteCache<K, ?> r) { super(r); }
    }
 
+   @ClientListener(converterFactoryName = "simple-converter-factory")
```

Comment:
And wondering if these names could be placed in a constant somewhere

Reply:
I also noticed some formatting is due in those listener tests, but this is not related to the JIRA, I will send another PR later (master only)

Reply:
Ok! if jenkins is ok I will merge the 3 of them unless you need an additional review

Reply:
Thanks @karesti !